docs/source/conf.py

```diff
@@ -13,6 +13,7 @@
 import os
 import sys
 from datetime import date
+from lasy import __version__
```

Comment:
Extra space to trigger CI
```suggestion
from lasy import __version__

```